docs/topics/migrations.txt

```diff
@@ -534,6 +534,30 @@ database migrations such as ``__init__()``, ``deconstruct()``, and
 which reference the field exist. For example, after squashing migrations and
 removing the old ones, you should be able to remove the field completely.
 
+.. _moving-model-between-apps:
+
+Moving a model between apps
+===========================
+
+  .. versionadded:: 5.0
+
+Django seamlessly handles the process of transferring models from one app
+to another. This involves recognizing the moved model and automatically
+generating the necessary migrations. A table rename will happen unless the
+moved model in new app defines a ``Meta.db_table`` that matches the old one.
+Before running the ``makemigrations`` command, it is mandatory to specify
+the original table name in the Meta section of the model being moved. This
+prevents the recreation of the model's table. The process involves four steps:
+
+* Creating unmanaged moved model in new app.
+
+* Altering model options to make moved model unmanaged in old app.
+
+* Altering model options to make moved model managed in new app.
+
+* Deleting moved model from old app.
+
```

Comment:
I think this deserves a more detailed explanation -- in particular, explain *why* we do things, not just *what* we do. Something like:
"""
When we are going to move a model -- that is, delete a model in one app and create it anew in another -- we want to avoid dropping and recreating its database table (which would lose all data). To achieve this, we first create the new model, making it point to the already existing table. To make Django not attempt to create that table (which would fail), we mark it `managed=False` upon creation. After this, we momentarily have two models pointing at the same table. Now, to remove the old model without deleting the table, we mark it `managed=False` as well. Finally, we can remove the old model and mark the new one `managed=True`.
"""

I'm not marking the above as a suggestion, for two reasons:
- It has style issues; in particular, the Django documentation mostly avoids parentheses and m-dashes; and,
- The optional table rename operation is not mentioned (I'm actually not quite sure about its place in the order of operations).